### DISCUSSION TITLE
Tunnel Fire Simulation

### DISCUSSION THREAD

**Discussion by shamimamita** [firemodels/fds#12633] [General]
I am working on tunnel fire simulations. I have defined fine mesh near the fire source. After 2 or second run of my simulations it shows maximum air density 4.08kg/m3 , problem in mesh line 2. Anybody tell me what type of problem it is?

**Comment by drjfloyd** [comment]
Density will increase when you have a net addition of mass to the domain. Inputs that result in more mass in the same volume will increase density. Carefully review your inputs and make sure you are using the correct units and look for where you might be adding mass to sealed volumes.

**Comment by shamimamita** [comment]
Here is my code please help me where is the fault&HEAD CHID='Fire35Middle'/
&TIME T_END=300.0/
&DUMP DT_PL3D=10.0, DT_RESTART=300.0, DT_SL3D=0.25, WRITE_XYZ=.TRUE., PLOT3D_QUANTITY='MASS FRACTION', 'VOLUME FRACTION', 'HRRPUV', 'PRESSURE', 'TEMPERATURE', PLOT3D_SPEC_ID(1:2)='AIR', 'AIR'/

&MESH ID='MESH', IJK=360,10,12, XB=0.0,14.0,0.0,0.4,-0.25,0.25/

&REAC FUEL = 'PROPANE',
FORMULA='C3H8',
CO_YIELD=0.04,
SOOT_YIELD=0.07/
&SURF ID='Wall & Roof'/
&SURF ID='Front Glass'/

&SURF ID='FIRE SOURCE',
COLOR='RED',
HRRPUA=465,
ADIABATIC=.TRUE./

&SURF ID='Supply',
COLOR='GREEN',
VOLUME_FLOW=-0.07/

&OBST ID='Back Wall', XB=0,14.0,0.4,0.42,-0.25,0.25, COLOR='INVISIBLE', SURF_ID='Wall & Roof'/
&OBST ID='Fireproof Front Glass', XB=0,14.0,-0.01,0,-0.25,0.25, COLOR='INVISIBLE', SURF_ID='Front Glass'/
&OBST ID='Obstruction1', XB=6.65,6.75,0.15,0.25,-0.25,-0.23, COLOR='RED', SURF_ID='INERT'/
&OBST ID='Obstruction2', XB=7.25,7.35,0.15,0.25,-0.25,-0.23, COLOR='RED', SURF_ID='INERT'/
&OBST ID='Obstruction3', XB=6.95,7.05,0.15,0.25,-0.25,-0.23, COLOR='RED', SURF_ID='INERT'/
&OBST ID='Floor', XB=0.0,14.0,0.0,0.4,-0.25,-0.25, COLOR='INVISIBLE', SURF_ID='Wall & Roof'/

&VENT ID='Mesh Vent: MESH [XMAX]', SURF_ID='OPEN', XB=14.0,14.0,0.0,0.4,-0.25,0.25/
&VENT ID='Mesh Vent: MESH [XMIN]', SURF_ID='OPEN', XB=0.0,0.0,0.0,0.4,-0.25,0.25/
&VENT ID='Burner-1', SURF_ID='FIRE SOURCE', XB=6.65,6.75,0.15,0.25,-0.23,-0.23/
&VENT ID='Burner-2', SURF_ID='FIRE SOURCE', XB=7.25,7.35,0.15,0.25,-0.23,-0.23/
&VENT ID='Burner-3', SURF_ID='FIRE SOURCE', XB=6.95,7.05,0.15,0.25,-0.23,-0.23/
&VENT ID='VENT', SURF_ID='Supply', XB=6.975,7.025,0.175,0.225,0.25,0.25/
@shamimamita
shamimamita
Mar 16, 2024
Author

when I define my mesh like this it shows density error, I can't find where is the fault '&MESH ID='MESH', IJK=466,16,16, XB=0.0,14.0,0.0,0.4,-0.25,0.25/
&MESH IJK=428,50,50, XB=6.0,09.0,0.0,0.4,-0.25,0.25 /'. Another question when I understand my simulation is in steady situation that means when I pick results from my simulation. I am a beginner in this field.Please give some ideas.

**Comment by rmcdermo** [comment]
You have several problems with your input file.

1. Don't start numbers with a 0.
2. When you changed your XB, you did not also change the position of the OPEN VENT, which must be on an external boundary.
3. You don't need a FORMULA for PROPANE.

If the case still does not run after fixing all those things, then you issue is memory on your system.

**Reply by shamimamita** [reply]
Can you please red mark where I have to change?

**Reply by rmcdermo** [reply]
No, you should be able to find this.

**Reply by shamimamita** [reply]
OK, I am trying.

**Reply by drjfloyd** [reply]
If you are just learning FDS. Start with very simple problems. A box with open sides containing a fire. Then add complexity a little bit at a time. If you immeadiately go to a more complex input file where you have mulitple errors, it can be difficult to determine what is going wrong. If you start simple and slowly add complexity then if results are suddenly not as expected you know what inputs to look at.

Also you should get in the habit of looking at the .out file and viewing your geometry in Smokeview. Make sure what is there is what you intended in your inputs.

**Reply by shamimamita** [reply]
Thanks for your suggestions

**Reply by oberbauleiter** [reply]
What is the downside of starting numbers with a "0."? There are many snippets in the User Guide with leading zeroes.

**Reply by drjfloyd** [reply]
I am not sure what input @rmcdermo was objecting to with his first item. If you have the real number input one-tenth, the namelist reader does not care if you put in 0.1 but you wouldn't want to have 00.1.  If you had the integer input 10, I have never tried to see what happens if you typed 010. Not sure if that would work or not.  You do want to make sure you don't accidentally type a capital letter O rather than the number 0.

**Reply by rmcdermo** [reply]
Think I was nit-picking this:
```
&MESH IJK=428,50,50, XB=6.0,**09.0**,0.0,0.4,-0.25,0.25 /
```
Might work, but seems sloppy to me.

**Reply by drjfloyd** [reply]
I missed that in the line at the end.

**Reply by oberbauleiter** [reply]
Thank you both for reviewing this statement!

**Comment by shamimamita** [comment]
Can anyone please tell me Where the coordinate system sits in the tunnel?

**Reply by drjfloyd** [reply]
open your geometry in smokeview and do right click --> show/hide > labels -> axis

**Reply by shamimamita** [reply]
Thanks you

**Comment by shamimamita** [comment]
> open your geometry in smokeview and do right click --> show/hide > labels -> axis

**Comment by shamimamita** [comment]
I have done it but it shows only X axis, how can I see the three axis?

**Reply by drjfloyd** [reply]
Rotate the view

**Comment by MilanJose123** [comment]
I am currently pursuing my Master’s in Advanced Computational Methods in Civil Engineering at RWTH Aachen University, and I am also working as a student researcher at the Federal Highway Research Institute in Germany, focusing on tunnel fire safety.

As part of my responsibilities, I am tasked with conducting fire simulations using the Fire Dynamics Simulator (FDS)and Smokeview(SMV). However, I am encountering challenges in learning FDS effectively and would greatly appreciate any guidance you could provide to help me approach this in the best way possible.
If there are any specific training materials, particularly video tutorials, that could aid my understanding, I would be very grateful if you could point me in the right direction.

**Reply by Vojta-Salek** [reply]
I would recommend you to start with very very simple FDS cases. Most features can be tested in the provided example cases or in a domain of like 5x5x5 computational cells. Try to slowly get familiar with how to setup your simulations, how to generate and check you results. User's guide (and eventually Technical guide) will be your best friends, do not skip reading it and do not expect to understand everything on the first reading. 

If you run into troubles, try to simplify your model and read relevant parts of user's guide again. If that does not help, you can always ask here. People are willing to help, but they need to be provided with enough data and with the right question.

Keep in mind that FDS does a lot of decisions for you. It is your responsibility to be aware of these default settings, but it takes time to explore everything. Step by step.

By the way I am not aware of any video tutorials, if you find some, please tag me here. Thanks!

**Reply by oberbauleiter** [reply]
I would like to add that while this is a viable way to learn how to operate the model, you also need a decent knowledge on the physical and chemical processes that go along with fires and their effects on the tunnel structure. I'm not sure if this part of your curriculum. If not, and if you want to assess if your results are valid, I'm afraid you will have to dig deeper into textbooks as well as published case studies. There are several suggestions given in the official guides.
